Client/Frontend/Contextual Hint/ContextualHintCopyProvider.swift

```diff
@@ -68,9 +68,12 @@ struct ContextualHintCopyProvider: FeatureFlaggable {
             let shouldShowNew = featureFlags.isFeatureEnabled(.copyForToolbar, checking: .buildOnly)
 
             if let arrowDirection = arrowDirection,
-               arrowDirection == .up,
                shouldShowNew {
-                descriptionCopy = CFRStrings.Toolbar.SearchBarTopPlacement
+                switch arrowDirection {
```

Comment:
nit:  To make it more readable and testable we could move this logic to a separate private func that returns the string giving `shouldShowNew` and `arrowDirection`

Reply:
Readable for sure agree, and I can adjust for this PR. But testable - Nimbus will get in the way of that right? At least for now?

Reply:
I was thinking on testing the function to begin with so we can pass the value for `shouldShowNew` without Nimbus